src/bundle/Controller/ContentController.php

```diff
@@ -331,4 +333,57 @@ public function previewAction(
             'versionNo' => $versionNo ?? $content->getVersionInfo()->versionNo,
         ]);
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @throws AdminInvalidArgumentException
+     * @throws \InvalidArgumentException
+     * @throws TranslationInvalidArgumentException
+     * @throws APIRepositoryInvalidArgumentException
+     * @throws UnauthorizedException
+     * @throws InvalidOptionsException
+     */
+    public function updateMainTranslationAction(Request $request): Response
+    {
+        $form = $this->formFactory->updateMainTranslation();
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
```

Comment:
If I remember correctly using `$form->isValid()` here will disable form errors to notifications mapping. It should be done by `SubmitHandler`. Please confirm I'm correct and fix if needed.